sktravel.py

```diff
@@ -37,10 +38,12 @@ def send_output(self, nick, dep, dest, date, result=None):
                 time = obj.lines[0].departure
             elif hasattr(result[0], 'drives') and len(obj.drives) >= 1:
                 time = result[0].drives[0].begin_time
+                decode_msg = False
             searched[nick] = [dep, dest, time, date]
 
-        out = unicode(out.strip(codecs.BOM_UTF8), 'utf-8')
-        return out.encode('utf-8')
+        if decode_msg:
```

Comment:
@Adman would it be possible to provide a of reasoning, as to why this works/is necessary?